pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -980,6 +980,8 @@ func buildDefaultPassthroughCluster() *apiv2.Cluster {
 		Type:           apiv2.Cluster_ORIGINAL_DST,
 		ConnectTimeout: 1 * time.Second,
 		LbPolicy:       apiv2.Cluster_ORIGINAL_DST_LB,
+		// Disable keep alive
+		MaxRequestsPerConnection: &types.UInt32Value{Value: 1},
```

Comment:
This has huge impact on performance. At least protect it with a flag/feature, we can't hardcode it to 1.

@mandarjog